site/content/entry/yuzu-progress-report-jun-2020/index.md

```diff
@@ -0,0 +1,174 @@
+    "./octopath-fixed.png"
+  >}}
+  
+&nbsp;
+   
+{{< single-title-imgs
+    "No more T-posing (Super Smash Bros Ultimate)"
+    "./ssbu-tpose.png"
+    "./ssbu-fixed.png"
+  >}}
+
+[Volca](https://github.com/VolcaEM) [stubbed an input instruction](https://github.com/yuzu-emu/yuzu/pull/4032) allowing `Minecraft: Nintendo Switch Edition` to go in-game.
+
+{{< imgs
+    "./minecraft.png| Don't dig too deep! (Minecraft: Nintendo Switch Edition)"
+  >}}
+  
+Not all games have to be in 3D. [Morph](https://github.com/Morph1984) [replaced the default texture filtering behaviour](https://github.com/yuzu-emu/yuzu/pull/4081) to achieve crisp graphics on 2D games like `Undertale`.
+
+{{< single-title-imgs
+    "Crisp, just the way I like it"
+    "./undertale-before.png"
+    "./undertale-after.png"
+  >}}
+  
+Last on our bug fixes section. Old AMD GPUs from 2012 lack the OpenGL `GL_EXT_texture_shadow_lod` extension, leading to shaders that failed to build in `The Legend of Zelda: Breath of the Wild`. As described [here](https://github.com/KhronosGroup/SPIRV-Cross/issues/207), by using the [textureGrad function](https://github.com/yuzu-emu/yuzu/pull/4129), [Morph](https://github.com/Morph1984) achieved similar results, improving compatibility with such old devices.
+
+## Vulkan improvements
+
+`OCTOPATH TRAVELER` got some Vulkan love thanks to [Rodrigo](https://github.com/ReinUsesLisp). By implementing a [missing image format](https://github.com/yuzu-emu/yuzu/pull/4033) and [storage texels and atomic image operations](https://github.com/yuzu-emu/yuzu/pull/4034) the game can now reach in-game status in Vulkan, and with some very beautiful visuals!
+
+{{< imgs
+    "./octopath.png| This is how nostalgia makes us remember the SNES era (OCTOPATH TRAVELER)"
+  >}}
+
+[Rodrigo](https://github.com/ReinUsesLisp) also worked on [multiple](https://github.com/yuzu-emu/yuzu/pull/4110) [performance](https://github.com/yuzu-emu/yuzu/pull/4111) optimizations for the Vulkan API. This results in lower frametimes and higher framerates on average.
+
+## User Interface changes
+
+An old misconception our users had with yuzu had to do with the `Internal Resolution` setting in the graphics settings. This was carried over from [Citra's](https://citra-emu.org/) UI, but it wasn't coded to do anything in yuzu. So, lots of users set it up at higher values expecting an image quality increase that could never happen. [Morph](https://github.com/Morph1984) decided it was time to [remove it for good](https://github.com/yuzu-emu/yuzu/pull/3966), and so he did.
+
+{{< imgs
+    "./res-scaler-removed.png| Until we meet again, Internal Resolution!"
+  >}}
+
+Thanks to users requesting it, [Volca](https://github.com/VolcaEM) added [Open Mods](https://github.com/yuzu-emu/yuzu/pull/4136) [FAQ and Quickstart](https://github.com/yuzu-emu/yuzu/pull/4166) shortcuts on yuzu's interface, a great way to help people just starting to use the emulator, you just need to click on Help!
```

Comment:
Change `a great way to help people just starting to use the emulator, you just need to click on Help!`

to

```This is a great way for users to find helpful information on how to use the emulator — you just need to click on `Help`!```